src/parser/jobs/drg/modules/Buffs.tsx

```diff
@@ -135,7 +135,7 @@ export default class Buffs extends Analyser {
 		this.checklist.add(new Rule({
 			name: <Trans id="drg.buffs.checklist.name">Keep {this.data.statuses.POWER_SURGE.name} up</Trans>,
 			description: <Trans id="drg.buffs.checklist.description">
-				<ActionLink {...this.data.actions.DISEMBOWEL}/> and <ActionLink action="COERTHAN_TORMENT" /> grant <StatusLink status="POWER_SURGE" /> which provides a 10% boost to your personal damage and should always be kept up.
+				<ActionLink {...this.data.actions.DISEMBOWEL}/> and <ActionLink action="SONIC_THRUST" /> grant <StatusLink status="POWER_SURGE" /> which provides a 10% boost to your personal damage and should always be kept up.
```

Comment:
For consistency's sake, can you change `<ActionLink {...this.data.actions.DISEMBOWEL}/>` to `<ActionLink action="DISEMBOWEL" />`? Seems kind of silly to have both flavors of `ActionLink` keying in a single string.

Reply:
hmm who could've possibly gotten lazy when the new action link format came out (it was me). fixed!